src/simulate/event_transformer/Transformer.py

```diff
@@ -58,11 +58,13 @@ def change_course_info(self, user):
             target = self.event['target']
             object['name'] = 'Lecture recorded on ' + utils.get_current_date()
             video_player = random.choice(a_course_info[utils.PROPS_VIDEO_PLAYER])
-            object['id'] = video_player
+            video_id = video_player[utils.PROPS_USER_ID]
+            object['id'] = video_id
+            object['name'] = video_player[utils.PROPS_PLAYER_NAME]
```

Comment:
With respect to your comment https://github.com/tl-its-umich-edu/caliper-event-analysis/pull/14/commits/11e53b3af07319a9240f708bb62bdf5192f4dd67#r148546687 , I forgot to delete the line `object['name'] = 'Lecture recorded on ' + utils.get_current_date()` but when the application ran it output the correct JSon as expected. The`object['name']` on line 59 value is over written by value in Line line 63.
I will send you the all the Json that is emitted to the endpoint in a bit 
```
{
    "sensor": "https://example.edu/sensors/1",
    "sendTime": "2017-11-02T15:18:29.406Z",
    "dataVersion": "http://purl.imsglobal.org/ctx/caliper/v1p1",
    "data": [
        {
            "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
            "id": "urn:uuid:3773c3fe-2b3b-4508-9dda-95c7e6961f52",
            "type": "MediaEvent",
            "actor": "https://mcommunity.umich.edu/#profile:user003",
            "action": "Resumed",
            "object": {
                "id": "https://leccap.engin.umich.edu/leccap/viewer/r/DOUPtp",
                "name": "kinetic energy",
                "type": "VideoObject",
                "duration": "PT1184.000S",
                "isPartOf": {
                    "id": "https://leccap.engin.umich.edu/leccap/site/ziiq5uzrd9k06e6vjew",
                    "type": "WebPage",
                    "name": "EECS 370",
                    "dateCreated": "2017-10-01T06:41:17.000Z"
                }
            },
            "target": {
                "id": "https://leccap.engin.umich.edu/leccap/viewer/r/DOUPtp#type=MediaLocation",
                "type": "MediaLocation",
                "name": "Lecture recorded on 10/01/2017",
                "dateCreated": "2017-10-01T19:08:04.000Z",
                "dateModified": "2017-10-01T21:29:03.000Z",
                "currentTime": "PT930.407S",
                "isPartOf": "https://leccap.engin.umich.edu/leccap/viewer/r/DOUPtp"
            },
            "eventTime": "2017-11-02T15:18:29.406Z",
            "edApp": "https://leccap.engin.umich.edu/leccap/",
            "session": {
                "id": "urn:umich:engin:leccap:session:E56DFF2D155bcaE182d",
                "type": "Session",
                "user": "https://mcommunity.umich.edu/#profile:user003",
                "startedAtTime": "2017-10-02T13:45:13.000Z"
            },
            "group": {
                "id": "https://leccap.engin.umich.edu/leccap/site/ziiq5uzrd9k06e6vjew#CourseOffering",
                "type": "CourseOffering",
                "courseNumber": "EECS 370",
                "academicSession": "FA 2017"
            },
            "membership": {
                "id": "https://leccap.engin.umich.edu/leccap/site/ziiq5uzrd9k06e6vjew#Membership&member=user003",
                "type": "Membership",
                "member": "https://mcommunity.umich.edu/#profile:user003",
                "organization": "https://leccap.engin.umich.edu/leccap/site/ziiq5uzrd9k06e6vjew#CourseOffering",
                "roles": [
                    "Learner"
                ],
                "status": "Active"
            },
            "federatedSession": {
                "id": "urn:instructure:canvas:umich:session:40B25Ab9CD161E14a9a9dFC5ECABf28403edFd",
                "type": "LtiSession",
                "user": "https://mcommunity.umich.edu/#profile:user003",
                "messageParameters": {
                    "custom_canvas_course_id": 184567,
                    "custom_canvas_user_id": "user003_cid",
                    "lis_course_offering_sourcedid": 216004546201101,
                    "lis_person_sourcedid": "user003_sisid",
                    "resource_link_id": "9a17100e4c806512a4a34b20d61dd8786d2c9826"
                }
            }
        }
    ]
}
```